packages/gestalt/src/RadioGroupButton.js

```diff
@@ -1,6 +1,7 @@
 // @flow strict
 import { type AbstractComponent, forwardRef, type Node as ReactNode, useState } from 'react';
 import classnames from 'classnames';
+import Badge from './Badge';
```

Comment:
Can we add an example in the Docs so we can see how it looks like?

Reply:
Done